frontends/web/src/routes/account/guide.tsx

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright 2022 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useTranslation } from 'react-i18next';
+import { IAccount, Coin, ISigningConfigurationList } from '../../api/account';
+import { Entry } from '../../components/guide/entry';
+import { Guide } from '../../components/guide/guide';
+import { isBitcoinBased, isBTCScriptType } from './utils';
+
+type Props = {
+  account: IAccount;
+  accountInfo?: ISigningConfigurationList;
+  coin?: Coin;
```

Comment:
call this `unit` or `coinUnit`, and the type is `string` - the unit is not the same as the coin code.